kilkari/src/main/java/org/motechproject/nms/kilkari/web/MctsBeneficiaryImportController.java

```diff
@@ -76,7 +76,7 @@ public void importMotherData(@RequestParam MultipartFile csvFile) {
             throw e;
         } catch (Exception e) {
             logError(csvFile.getOriginalFilename(), "/kilkari/mother/import", e);
-            //todo: why are we not throwing here and throwing in importChildData below?
+            throw new CsvImportException("An error occurred during CSV import", e);
```

Comment:
it should be contained as an internal exception in e. We are required to throw a csvImportException by the spec... but none of that matters anymore... since csv is a dinosaur now. this change was more to follow the pattern below with the child import.